optimize/backend/src/main/java/io/camunda/optimize/service/db/repository/es/ProcessInstanceRepositoryES.java

```diff
@@ -70,14 +71,19 @@ public void deleteByIds(
         BulkRequest.of(
             b ->
                 b.operations(
-                    o -> {
-                      processInstanceIds.forEach(
-                          id ->
-                              o.delete(
-                                  d ->
-                                      d.id(id).index(esClient.addPrefixesToIndices(index).get(0))));
-                      return o;
-                    }));
+                    processInstanceIds.stream()
```

Comment:
Please get the C7 fixes to this PR as well

Reply:
Planning to - this one isnt ready for review yet